### ISSUE TITLE
fits.ImageHDU.scale()

### ISSUE THREAD

**Issue by ppmim** [astropy/astropy#6674]
I think ImageHDU.scale() function has a bug (see below). It does not happen under Pyfits3.3:

Under Python 2.7.6:
```python
>>> import numpy as np
>>> import astropy
>>> import astropy.io.fits as fits
>>> astropy.__version__
u'1.3.3'
>>> np.__version__
'1.7.1'
>>> a = np.array([2.0,3.0,4.0])
>>> hdu = fits.ImageHDU(data=a)
>>> hdu.scale("float32")
_zero None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/panic/.local/lib/python2.7/site-packages/astropy-1.3.3-py2.7-linux-x86_64.egg/astropy/io/fits/hdu/image.py", line 460, in scale
    bzero=bzero, blank=None)
  File "/home/panic/.local/lib/python2.7/site-packages/astropy-1.3.3-py2.7-linux-x86_64.egg/astropy/io/fits/hdu/image.py", line 522, in _scale_internal
    np.add(self.data, -_zero, out=self.data, casting='unsafe')
TypeError: bad operand type for unary -: 'NoneType'
```

**Comment by saimn** [comment]
Looks similar to #6399, which was fixed by #5955 in v2.0.1. Could you try with this version ?

**Comment by ppmim** [comment]
Yes, i have tried, and it is solved in v2.0.1. Thanks !

**Comment by MSeifert04** [comment]
@ppmim Thank you. 

I'm closing the issue now :)
